administration/src/bp-modules/applications/VerificationsQuickIndicator.tsx

```diff
@@ -1,51 +1,57 @@
-import { Tooltip } from '@blueprintjs/core'
-import React, { memo } from 'react'
+import { Container, Stack, Tooltip, useTheme } from '@mui/material'
+import React, { forwardRef, memo } from 'react'
 import { useTranslation } from 'react-i18next'
-import styled from 'styled-components'
 
 import type { Application } from './ApplicationsOverview'
 import VerificationIndicator from './components/VerificationIndicator'
 import { VerificationStatus } from './constants'
 import { getVerificationStatus } from './utils'
 
-export const UnFocusedDiv = styled.div`
-  display: flex;
-  align-items: center;
-  flex: 1;
-  :focus {
-    outline: none;
-  }
-  height: 25px;
-`
-
-const VerificationQuickIndicator = memo(({ verifications }: { verifications: Application['verifications'] }) => {
-  const verificationStati = verifications.map(getVerificationStatus)
+const ToolTipContent = forwardRef<HTMLDivElement, unknown>((p, ref) => {
   const { t } = useTranslation('applicationsOverview')
+  const theme = useTheme()
+
+  return (
+    <Container
+      ref={ref}
+      sx={{
```

Comment:
❓Just a general question. Why do we not use the mui `styled` api here?
I mean if we want to style just a particular part of a complex component we have to need the particular props but i general i prefer the styled components style, since the readability is way better

Reply:
When making a one-off customization of a component, I generally use the `sx` prop. For components that get reused or I want to externalize, I would probably use the `styled()` approach. I tend to follow these guidelines:

https://mui.com/material-ui/customization/how-to-customize/#overriding-nested-component-styles

Though I'm a bit annoyed that there seems to be no API to combine both ways (make a component with a styled top-level element and provide e.g. custom `slotProps`) and that the `styled()` API are slightly different in functionality: https://mui.com/system/styled/#difference-with-the-sx-prop

Reply:
Hm my personal preference would be to use `styled()` if its possible even if the components are not reused because i find it better to read and it's clearer when to use what.
In all other cases where particular parts via props have to be styled for sure this way has to be used.
Thats also how we did it in other projects so far afaics.

But maybe others of the team have another opinion

Reply:
ok i'm fine to keep that for now.
yes sure we shouldn't mix styled apis